src/sonic-pit/pit-sysdiag/src/errcode.py

```diff
@@ -0,0 +1,251 @@
+    EBMC21003 = 21003                     # BMC emmc test failed
+    EBMC21004 = 21004                     # BMC i2c test failed
+    EBMC21005 = 21005                     # BMC rtc test failed
+    EBMC21006 = 21006                     # BMC gpio test failed
+    EBMC21007 = 21007                     # BMC spi test failed
+    EBMC21008 = 21008                     # BMC mdio test failed
+
+    ESPI22001 = 22001                     # spi bus devices access failed
+
+    ESW23001 = 23001                      # switch chip tcam fault
+
+    EKR24001 = 24001                      # kr not recognize
+    EKR24002 = 24002                      # kr link down
+    EKR24003 = 24003                      # kr drop packet
+
+    EDISK25001 = 25001                    # disk capacity not enough
+    EDISK25002 = 25002                    # disk size not enough
+
+    ETIMEZONE26001 = 26001                # time zone not match
+
+    EFWVERSION27001 = 27001               # bios version not match
+    EFWVERSION27002 = 27002               # pit version not match
+    # sonic node running version is lower than sonic node baseline version.
+    EFWVERSION27003 = 27003
+    # pit node running version is lower than sonic node baseline version.
+    EFWVERSION27004 = 27004
+
+    ENET28001 = 28001  # link speed not match
+    ENET28002 = 28002  # ping test loss packet
+
+    EFWBMCUPGRADE29001 = 29001  # bmc upgrade [slave] fail
+    EFWBMCUPGRADE29002 = 29002  # bmc upgrade [master] fail
+    EFWBMCUPGRADE29003 = 29003  # bmc set boot flash to [slave] fail
+    EFWBMCUPGRADE29004 = 29004  # bmc set boot flash to [master] fail
+    EFWBMCUPGRADE29005 = 29005  # bmc version not match
+    EFWBMCUPGRADE29006 = 29006  # bmc flash match
+    EFWBMCUPGRADE29007 = 29007  # get bmc version fail
+    EFWBMCUPGRADE29008 = 29008  # get bmc flash fail
+    EFWBMCUPGRADE29009 = 29009  # ping bmc fail,time out
+    EFWBMCUPGRADE29010 = 29010  # get bmc upgrade file fail
+
+    ESSDSTRESS30001 = 30001  # ssd total size not match.
+    ESSDSTRESS30002 = 30002  # fio ssd stress test fail.
+    ESSDSTRESS30003 = 30003  # ssd read counts error.
+
+    ENETSTRESS31001 = 31001  # cpu ip not legal.
+    ENETSTRESS31002 = 31002  # ip ping test loss packet.
+    ENETSTRESS31003 = 31003  # iperf tool not exist.
+    ENETSTRESS31004 = 31004  # iperf test loss packet.
+
+    EBIOSME32001 = 32001  # BIOS recovery open fail.
+    EBIOSME32002 = 32002  # BIOS recovery close fail.
+
+    ESVID33001 = 33001  # high voltage not match.
+    ESVID33002 = 33002  # low voltage not match.
+    ESVID33003 = 33003  # standerd voltage not match.
+
+    ESNAKE34001 = 34001  # snake sdk enter demon mode fail.
+    ESNAKE34002 = 34002  # snake port status error.
+    ESNAKE34003 = 34003  # snake port pkt conut fail.
+
+    ESSH35001 = 35001   # [ remote ] connect fail ,PASSWORD OR HOSTNAME is wrong
+    # [ remote ] connect timeout ,PASSWORD OR HOSTNAME is wrong
```

Comment:
move to next line?

Reply:
Done, move the comment to next line, after the error code.